src/test/java/com/gooddata/featureflag/FeatureFlagServiceAT.java

```diff
@@ -87,6 +87,22 @@ public void deleteProjectFeatureFlag() throws Exception {
         // intentionally don't check the flag is really deleted because of eventual consistency of this API
     }
 
+    @Test(groups = "featureFlag", dependsOnMethods = "deleteProjectFeatureFlag")
```

Comment:
Why does it depend on method touching different flag? What about `updateProjectFeatureFlag`?

Reply:
And one question, when i can test it agains my PI, what is projectToken and warehouseToken?
Thnx